api consistency

nteetor opened the pull request:
## updates

* input functions now try to follow shiny's argument conventions; selecting, disabling, and any other item related markup is handled by indicating the value rather than with a separate helper function. 
* `tableThruput` is revised
* `formInput` has been fixed, other form related inputs are removed
* most input helper functions are gone, helper functions like `accordionPanel` are still around

waynenilsen commented:
I am reading through the docs now and I do see that there is no need to include the beginning `/` but that was not the case for my setup

nteetor commented:
Hello Wayne, thank you for bringing this to my attention. Rather than patch the documentation I am going to fix the uri matching. I'll make sure to tag you in the commit.
Nate

Commit:
beginning work on checkbox, radio consistency

Commit:
making checkbox and radio consistent with api, improved JS, added update* functions, small improvements

Commit:
updated js packages

Commit:
small patches to js files

Commit:
cleaned up id reference in radio, finished toy docs example

Commit:
fixing buttonInput and dropdownInput calls

Commit:
updating references to buttonInput, listGroupInput, and dropdownInput

Commit:
attempt at new tooltip functionality

Commit:
tweeks to table thruputs

Commit:
javasript updates

Commit:
updating checkbox re #12

Commit:
removed outdated function

Commit:
namespace rebuild

Commit:
finalized checkbox conversion

Commit:
minor typo

Commit:
radioInput updated for consistency

Commit:
removed checkbox `inline` argument

Commit:
how long has this been here?

Commit:
patched radio values update functionality

Commit:
listGroupInput under repair; sets better example than checkbox or radio

Commit:
minor change

Commit:
allow filtering of radio enabled and disabled

Commit:
limit validate to "valid", "warning" and "danger", fixed list group input

Commit:
button input count to clicks

Commit:
docs match new functionality, values defaults to labels

Commit:
enable, disable working for radio, doc updates

Commit:
enable, disable working for radio, doc updates

Commit:
checkbox input value defaults to label, update fixed

Commit:
merge

Merge branch 'api-consistency' of https://github.com/nteetor/dull into api-consistency

# Conflicts:
#	inst/www/js/dull.min.js
#	man/radioInput.Rd

Commit:
added enable and disable list group

Commit:
added increment list group functionality

Commit:
dropdown input undergoing consistency transformation

Commit:
button update, removed button value

Commit:
typo fix

Commit:
added checkbox to inputs family

Commit:
Merge branch 'api-consistency' of https://github.com/nteetor/dull into api-consistency

Commit:
fixed button update not preserving outline

Commit:
dropdownInput can now be disable

Commit:
enable dropdown inputs, disabling now triggers change & removes value

Commit:
dropdown docs touch up

Commit:
updated inputs to work better inside formInput

Commit:
added formInput

Commit:
added formInput

Commit:
merge formInput

Merge branch 'api-consistency' of https://github.com/nteetor/dull into api-consistency

# Conflicts:
#	R/forms.R

Commit:
fixed reactivity of inputs inside formInputs

Commit:
group inputs have single value, button triggers reactive, working on dropdown

Commit:
add dropdown addon to group input, button addon not working

Commit:
fixed bad example, group input was not broken

Commit:
updated badgeOutput for api consistency

Commit:
small form input JS update

Commit:
clean up of unecessary css, js files

Commit:
fixed disappearing badges by adding dummy renderError function

Commit:
limit interactivity of non-reactive list groups

Commit:
updated select input

Commit:
forgotten update of docs

Commit:
formInputs scope variables, form element reactive inputs return null, access real values through form id, re #2

Commit:
removed misc file

Commit:
textual input updates on `input` event, minor form input improvements

Commit:
readme update

Commit:
better argument names, selectInput `multiple` added, height set to 100% when TRUE re #17

Commit:
improving tableThruput, removed breadcrumbs

Commit:
cleaned up js bundling, added css bundling

Commit:
fixed css gulp, added back missing font awesome files

Commit:
added column (de)selector to table thruput

Commit:
table row and column selection updates

Commit:
depend on jsonlite for table input

Commit:
table input now properly returns data frame

Commit:
improved table thruput row and column selection

Commit:
re #18, table thruput has been drastically simplified

Commit:
table thruput doc minor change

Commit:
fixed bugs in collapse, reworked collapse re #5 and #14

Commit:
hoping to fix wacky gulp issues

Commit:
update stream output, #1

Commit:
update stream output, #1

Commit:
merge

Merge branch 'api-consistency' of https://github.com/nteetor/dull into api-consistency

# Conflicts:
#	R/stream.R
#	man/streamOutput.Rd

Commit:
removed unused argument

Commit:
use del over gulp clean

Commit:
documentation touch up, prepping for pull to master

Commit:
added missing class

Commit:
modal and alert logic/generation moved to JS

Commit:
removed unused file

Commit:
small tune ups before pull into master

Commit:
Merge branch 'master' into api-consistency
